paket.dependencies

```diff
@@ -46,15 +47,15 @@ group Google
     framework: netstandard2.0
     nuget FSharp.Core >= 4.7
     nuget Google.Protobuf ~> 3
-    nuget Taskbuilder.fs ~> 2
+    nuget Ply
     nuget Oryx
 
 group Test
     source https://www.nuget.org/api/v2
     storage: none
 
     nuget FSharp.Core >= 4.7
-    nuget coverlet.msbuild 2.5.1
+    nuget coverlet.msbuild ~> 3
```

Comment:
Note that this is the reason for the code coverage being reduced. We will need to disable coverage requrement on GitHub to get it merged